sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/TablesModelHelper.java

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables.implementation;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.data.tables.implementation.models.TableResponseProperties;
+import com.azure.data.tables.models.TableEntity;
+import com.azure.data.tables.models.TableItem;
+
+import java.time.OffsetDateTime;
+import java.util.Objects;
+
+/**
+ * Used to access internal methods on models.
+ */
+public final class TablesModelHelper {
+    private static EntityAccessor entityAccessor;
+    private static ItemCreator itemCreator;
+
+    static {
+        // Force classes' static blocks to execute
+        try {
+            Class.forName(TableEntity.class.getName(), true, TableEntity.class.getClassLoader());
+            Class.forName(TableItem.class.getName(), true, TableItem.class.getClassLoader());
+        } catch (ClassNotFoundException e) {
+            AssertionError err = new AssertionError("Failed to initialize TablesModelHelper dependency classes.", e);
+            new ClientLogger(TablesModelHelper.class).logThrowableAsError(err);
```

Comment:
I believe the throw should be here

Reply:
That's how it was before but it causes a SpotBugs error. Apparently, a throw block inside a static initializer must throw an AssertionError. However the return type of `logThrowableAsError` is Throwable. So it complains about an unchecked cast. Since `logThrowableAsError` just takes a throwable, logs it, and returns the same throwable, I felt this was a better way rather than suppressing the unchecked cast warning.